features/admin.connections.v1/components/create/custom-authentication-create-wizard.tsx

```diff
@@ -928,6 +935,27 @@ const CustomAuthenticationCreateWizard: FunctionComponent<CustomAuthenticationCr
                 data-componentid={ `${_componentId}-create-wizard-endpoint-uri` }
                 width={ 15 }
             />
+            <FormSpy
+                onChange={ ({ values }: { values: EndpointConfigFormPropertyInterface }) => {
+                    if (values?.endpointUri?.startsWith("http://")) {
+                        setIsHttpEndpointUri(true);
+                    } else {
+                        setIsHttpEndpointUri(false);
+                    }
+                } }
+            />
+            { isHttpEndpointUri && (
+                <Alert
+                    severity="warning"
+                    className="endpoint-uri-alert"
+                    data-componentid={ `${_componentId}-endpoint-uri-alert` }
+                >
+                    <Trans
+                        i18nKey={ t("actions:fields.endpoint.validations.notHttps") }
+                    >
```

Comment:
If we are using trans component, lets add the default text here.

If the rendering text doesn't have formatting, we can use t(i18n_key)

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/7563/commits/7b54f298633dde6597c117612ecd1f4dd7c7232d